.jcheck/conf

```diff
@@ -39,3 +39,6 @@ files=.*\.java$|.*\.yml$|.*\.gradle$|.*.\txt$
 
 [checks "reviewers"]
 reviewers=1
+
+[checks "merge"]
+message=Merge jdk
```

Comment:
Do we really want this check for Skara? :)

Reply:
Haha, some deubbing leftovers, I will clean up :smiley_cat: